src/main/webui/src/app/components/content/group/GroupEdit.jsx

```diff
@@ -14,8 +14,306 @@
+              e.preventDefault();
+              removeConstituent(item);
+            }}>&#x2718;</a>
+            {
+              index !== 0 && <a href="" title="Promote priority" className="inline-cp-action" onClick={e => {
+                e.preventDefault();
+                promote(item);
+              }}>&uarr;</a>
+            }
+            {
+              index !== 0 && <a href="" title="Top priority" className="inline-cp-action" onClick={e => {
+                e.preventDefault();
+                top(item);
+              }}>&#x219f; </a>
+            }
+            {
+              index !== store.constituents.length - 1 && <a href="" title="Demote priority" className="inline-cp-action" onClick={e => {
+                e.preventDefault();
+                demote(item);
+              }}>&darr;</a>
+            }
+            {
+              index !== store.constituents.length - 1 && <a href="" title="Bottom priority" className="inline-cp-action" onClick={e => {
+                e.preventDefault();
+                bottom(item);
+              }}>&#x21a1;</a>
+            }
+          </div>
+        </div>
+        </li>)
+      }
+    </ol>
+    <ol className="right-half detail-value detail-edit-list">
+      <lt><label>Available:</label><span className="hint">(click to add to constituents)</span></lt>
+      {
+        currentAvailable && currentAvailable.map(item => <li key={`available-${item}`}><div className="available">
+          <a href="" title="Add to constituents" className="surround-cp-action" onClick={e => {
+            e.preventDefault();
+            addConstituent(item);
+          }}>
+            <div className="inline value">{item}</div><span className="inline-cp">+</span>
+          </a>
+        </div>
+        </li>)
+      }
+    </ol>
+  </div>;
+};
+
+EditConstituents.propTypes = {
+  store: PropTypes.object,
+  currentAvailable: PropTypes.array,
+};
 
 export default function GroupEdit() {
-  return <div>This is not implemented yet!</div>;
+  const [store, setStore] = useState({"packageType": "maven", "type": "group", "constituents": []});
+  const [available, setAvailable] = useState([]);
+  const location = useLocation();
+  const {packageType, name} = useParams();
+  const {
+    register,
+    reset,
+    trigger,
+    handleSubmit,
+    formState: {errors}
+  } = useForm();
+
+  const path = location.pathname;
+  const mode = path.match(/.*\/new$/u) ? 'new' : 'edit';
+  useEffect(() => {
+    if (mode === 'edit') {
+      const fetchStore = async () => {
+        // get Store data
+        const res = await storeRes.get(packageType, "group", name);
+        if (res.success) {
+          const raw = res.result;
+          const storeView = Utils.cloneObj(raw);
+          storeView.disabled = raw.disabled === undefined ? false : raw.disabled;
+          // get available Store data
+          const availableRes = await statsRes.getAllEndpoints();
```

Comment:
Forgot one thing. This API will return all store data for all package types. We should filter out the stores for current package type.

Reply:
removed other packageTypes and added dynamic update on changing the packagetype for new package